test/unit/org/apache/cassandra/tools/nodetool/InvalidateCIDRPermissionsCacheTest.java

```diff
@@ -118,10 +115,13 @@ public void testInvalidateSingleCidrPermission()
         // ensure cidr permission is cached
         role.hasAccessFromIp(ipAddr);
         assertThat(originalReadsCount).isEqualTo(getCidrPermissionsReadCount());
+        assertThat(((CassandraCIDRAuthorizer) DatabaseDescriptor.getCIDRAuthorizer()).has(ROLE_A)).isTrue();
```

Comment:
`"Invalidated the role role_a from CIDR permissions cache"` gets printed only when entry exists before invalidation. i.e, `assertThat(tool.getStdout()).contains("Invalidated the role role_a from CIDR permissions cache");` below at line 123 you added ensures that invalidation of a role was successful. So we do not need has(role) function.